backend/README.md

```diff
@@ -44,12 +44,21 @@ Run production build with a different env file.
 
 ### Environment Variable:
 
-In order to connect to the Plaid API, a .env was created in the backend.
-Your .env file should contain two values that were generated by
-the PLAID API, a PLAID_CLIENT_ID and a PLAID_SECRET.
-One can retrieve these values upon logging into the PLAID website and navigating to one's individual
-dashboard. It is important to let the .gitignore file know that these values CAN NOT be checked
-into Github. Additionally, three more envioronment variables should be configured to initially
-request a Link Token : PLAID_ENV, PLAID_PRODUCTS and PLAID_COUNTRY_CODES. The File
-backend\src\routes\BankingRoutes.ts imports the .env file and sets up the initial connection
-via its get endpoint.
+To connect to the Plaid API, you'll need to set up environment variables. Here's what you need to do:
+
+Create a .env file: Start by creating a .env file in your backend directory.
+
+Add Plaid API credentials: Inside the .env file, include the following values:
+
+PLAID_CLIENT_ID: This is your Plaid API client ID.
+PLAID_SECRET: Your Plaid API secret.
+These credentials are generated by the Plaid API and can be obtained by logging into your Plaid account and navigating to your individual dashboard. It's crucial to remember not to commit these credentials to GitHub. Make sure they are included in your .gitignore file.
+
+Configure additional variables: In addition to the client ID and secret, you'll need to set up three more environment variables for the initial request of a Link Token:
+
+PLAID_ENV: The environment to use (e.g., sandbox, development, production).
+PLAID_PRODUCTS: The products to access (e.g., transactions, auth, identity).
+PLAID_COUNTRY_CODES: The country codes for the institutions you want to support.
```

Comment:
This is incorrect, as they are hard-coded in BankingRoutes.ts. Please remove this section.

Reply:
Do you mean update?

Reply:
Read the next comment